src/main/resources/google_checks.xml

```diff
@@ -134,6 +134,11 @@
             <message key="name.invalidPattern"
              value="Parameter name ''{0}'' must match pattern ''{1}''."/>
         </module>
+        <module name="LambdaParameterName">
+            <property name="format" value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$"/>
+            <message key="name.invalidPattern"
+                     value="Lambda parameter name ''{0}'' must match pattern ''{1}''."/>
```

Comment:
@romani Do we really need message override for google? Should this just be the default message?

Reply:
@rnveach , a lot of naming Checks reuse `name.invalidPattern` - https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fnaming+filename%3Amessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22name.invalidPattern%22
in google style config "all" naming checks override general message to be a bit more precise. So in this Check override was done the same way.
I do not remember exact reason of such overrides, probably it was a problems to read violations in general report - http://checkstyle.sourceforge.net/reports/google-style/guava/ so we decided to make it more exact.
At least for now, it is better to be consistent. I am ok to have override here.

Reply:
ok.